Decred Wallet/table_view_cell/AccountDataCell.swift

```diff
@@ -62,18 +60,17 @@ class AccountDataCell: UITableViewCell, AccountDetailsCellProtocol {
             labelHDPathValue.text = "\(GlobalConstants.Strings.MAINNET_HD_PATH) \(account.Number)'"
         }
         
-        if (account.Number == INT_MAX) {
+        if account.Number == INT_MAX {
             defaultAccount.setOn(false, animated: false)
             defaultAccount.isEnabled = false
             hideAcount.setOn(false, animated: false)
             hideAcount.isEnabled = false
-        }else{
-            let hidden = UserDefaults.standard.bool(forKey: "hidden\(account.Number)")
+        }else {
+            let hidden = UserDefaults.standard.bool(forKey: "\(Settings.Keys.HiddenWallet)\(self.accountTmp.Number)")
```

Comment:
Why is `account` changed to `self.accountTmp`? Do they refer to the same variable? If so, one should suffice, I believe, the other can be deleted.